docs/Tutorials/Tutorial-design.md

```diff
@@ -0,0 +1,38 @@
+# Tutorial Design
+
+The purpose of a hack week is to expose participants to a broad range of data science tools and methods. With this in mind, we suggest the following guidelines when developing a tutorial:
+
+* Tutorials should be designed to address broad challenges in the data science community being served by the event. Speaking with community groups, reading blogs and attending workshops are good ways to connect with the most urgent data science challenges in a particular community. For example, the ICESat-2 hack week tutorial content was largely informed by members of the satellite's science team who were directly involved in releasing the data to the research community. 
+* Based on the needs of the community, and in consultation with other event planners, build an outline of desired learning outcomes.  

```

Comment:
By "other event planners", do you mean "other planners involved in hack week" or "planners of other events"?